source/MaterialXContrib/MaterialXMaya/README.md

```diff
@@ -8,62 +8,101 @@ This module contains code for a sample Maya plug-in which implements Maya surfac
+The plug-in defines two Maya nodes: `MaterialXSurfaceNode` and `MaterialXTextureNode`, accessible from the Hypershade under the _Surface_ and _2D Textures_ categories, respectively. A `MaterialXSurfaceNode` can be assigned to a shape via the [Marking Menu](https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2019/ENU/Maya-LightingShading/files/GUID-D41AF807-F7CB-447E-BACC-7F0867C14E8D-htm.html).
+
+A `MaterialXTextureNode` can be mapped to any shader. For example, you can map its OutColor attribute to the Color attribute of a Lambert, or the Specular Color of an aiStandardSurface. 
 
 Both node types expose the following attributes:
-*   `documentFilePath`: the file path to the MaterialX document (an XML file with extension .mtlx);
-*   `elementPath`: the path to the MaterialX element within the document to generate the shader for;
-*   `environmentRadianceMap`: the file path to the environment map to use for specular shading, defaults to a texture that ships with the plug-in;
-*   `environmentIrradianceMap`: the file path to the environment map to use for diffuse shading, defaults to a texture that ships with the plug-in.
+*   `documentFilePath`: the file path to the MaterialX document (an XML file with extension `.mtlx`).
+*   `elementPath`: the path to the MaterialX element within the document. 
+Use the format <material name/element name>. 
+*   `environmentRadianceMap`: the file path to the environment map to use for specular shading. Defaults to a texture that ships with the plug-in.
+*   `environmentIrradianceMap`: the file path to the environment map to use for diffuse shading. Defaults to a texture that ships with the plug-in.
+
+You can find the `environmentRadianceMap` and `environmentIrradianceMap` attributes in the Extra Attributes section of the Attribute Editor.
+
+The element name can be found in the shaderref element of the .mtlx document. For example, in `standard_surface_carpaint.mtlx`, the material name and element name are as follows:
+
+```
+<material name="Car_Paint">
+    <shaderref name="SR_carpaint" node="standard_surface">
+```
+    
+The element path is therefore: _Car_Paint/SR_carpaint_.
+
 
 Modifying any one of these attributes via GUI or scripting causes the shading node to refresh in the viewport.
 
 ## Maya Commands
 ### `createMaterialXNode`
 
-Creates one or many MaterialX nodes from the supplied MaterialX document. The command accepts the following flags:
+Creates one or many MaterialX nodes from the specified MaterialX document. The command accepts the following flags:
 
 ```MEL
    -d -documentFilePath  String
 ```
-
 Required argument: the path to a valid MaterialX document.
 
+> Note: A shader that is created via command is not listed in the Browser of the Hypershade (unlike those created via the Create tab of the Hypershade). It will, however, appear in the marking menu, from which you can assign it to your object. 
```

Comment:
```suggestion
> Note: A shader that is created via the command is not listed in the Browser of the Hypershade (unlike those created via the Create tab of the Hypershade). It will, however, appear in the marking menu, from which you can assign it to your object. 
```